Scripts/build-ohttpstubs.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
```

Comment:
Based on [OHHTTPStubs readme](https://github.com/AliSoftware/OHHTTPStubs#using-the-right-swift-version-for-your-project) - "CocoaPods will compile OHHTTPStubs with the right Swift Version matching the one you use for your project automatically"

So, it already has the logic to choose right parameters based on XCode version. Also, all swift code is placed to separate target, so we don't need to use it at all.

It's not clear why we need this script at all instead of just referencing the project in XCode